Source/SlashCmd/Main.lua

```diff
@@ -0,0 +1,50 @@
+local _, addonTable = ...;
+local zc = addonTable.zc;
+
+function Auctionator.SlashCmd.Initialize()
+  SlashCmdList["Auctionator"] = Auctionator.SlashCmd.Handler
+  SLASH_Auctionator1 = "/auctionator"
+  SLASH_Auctionator2 = "/atr"
+end
+
+function Auctionator.SlashCmd.Handler(input)
+  Auctionator.Debug.Message( 'Auctionator.SlashCmd.Handler', msg )
+
+  local command = {zc.words(input:lower())};
+
+  if #command == 0 then
+    return
+  end
+  if command[1] == "reset" or command[1]=="r" then
+    Auctionator.SlashCmd.Reset(command[2])
+  elseif (command[1] == "toggle" or command[1]=="t") then
+    Auctionator.SlashCmd.Toggle(command[2])
+  else
+    Auctionator.Utilities.Message("Unrecognized command")
```

Comment:
```lua
Auctionator.Utilities.Message("Unrecognized command: " .. command[1])
```
For when I stupidly type something not on the list and wonder what I did wrong. 😀